config/core/test-configs.yaml

```
@@ -3102,6 +3102,15 @@ test_configs:
 # Disabling to reduce load
 #      - v4l2-compliance-uvc
 
+  - device_type: mt8192-asurada-spherion-r0
+    test_plans:
+      - v4l2-decoder-conformance-h264
+      - v4l2-decoder-conformance-vp8
+      - v4l2-decoder-conformance-vp9
+    filters: &chromebook-decoders-filter
+      - passlist: {defconfig: ['videodec']}
+      - regex: {'tree': '(next|mainline|media)'}
```

As previously discussed, a potential improvement might be to narrow down the regex strings to avoid matching things like `mainlineXYZ`:
```suggestion
      - regex: {'tree': '^(next|mainline|media)$'}
```
There are however other filters with the same issue so maybe a follow-up PR to solve them all in one go would be good.  It's not currently causing any practical issues as there aren't any other tree names currently that would match those listed there (`net-next` doesn't match this regex).

---

Yes, I plan on submitting a separate PR to address all regex strings in the file.